### PR TITLE
backoffice: item details

### LINKED ISSUE #57: Back-office ui: implement mockups for item details
Implement item details page to show all fields for an item.
Depends on #63

### PR DESCRIPTION
**Pull request by topless**
- Resolved info in the item details page
- Details style improvement
- Closes #57

### CODE REVIEW

**Comment by topless** [comment]
screenshot 2018-11-22 at 13 41 19

**Comment by ntarocco** [comment]
Test failing because of:
```
invenio_app_ils/records/serializers/__init__.py:1 at module level:
        D104: Missing docstring in public package
```

File: invenio_app_ils/records/serializers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+from invenio_records_rest.serializers.json import JSONSerializer
+from invenio_records_rest.serializers.response import record_responsify
+from invenio_records_rest.schemas import RecordSchemaJSONV1
+
```
Comment (ntarocco) [inline invenio_app_ils/records/serializers/__init__.py:13]: missing doc string here

File: invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemMetadata/ItemMetadata.scss [hunk 2]
```diff
@@ -2,4 +2,7 @@
   .field-name {
     text-transform: capitalize !important;
```
Comment (kpsherva) [inline invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemMetadata/ItemMetadata.scss:3]: I have a general question: is the usage  of `!important` in css something react specific?

Comment (ntarocco) [reply invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemMetadata/ItemMetadata.scss:3]: Nope! Probably a [Semantic UI React issue](https://react.semantic-ui.com)!

Comment (topless) [reply invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemMetadata/ItemMetadata.scss:3]: @kprzerwa @ntarocco its a semantic issue indeed, the solutions are either styled components, I guess adjustments to the build system so we can configure way css is build, or we keep having our eye sockets bleeding with `!important` crap.

**Review by ntarocco — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Pull request merged**